spec/factories/slides.rb

```diff
@@ -33,6 +33,7 @@
     num_of_pages 1
     sequence(:created_at, 1) { |i| Time.now - i.days }
     downloadable true
+    tag_list "Sushi,Toro"
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->